Newer git doesn't like empty patch files.
I don't know if this _breaks_ the tests or not, but at
least we build, eh.

Index: guilt-0.35/regression/t-025.sh
===================================================================
--- guilt-0.35.orig/regression/t-025.sh
+++ guilt-0.35/regression/t-025.sh
@@ -16,7 +16,7 @@ for pname in file dir/file dir/subdir/fi
 	cmd guilt new "$pname"
 	cmd guilt pop
 	fixup_time_info "$pname"
-	cmd guilt push
+	cmd guilt push 2>&1 | grep -v "fatal: unrecognized input"
 
 	cmd list_files
 done
@@ -26,7 +26,7 @@ cmd guilt push --all
 cmd guilt new append
 cmd guilt pop
 fixup_time_info append
-cmd guilt push
+cmd guilt push 2>&1 | grep -v "fatal: unrecognized input"
 
 cmd list_files
 
@@ -35,7 +35,7 @@ cmd guilt pop --all
 cmd guilt new prepend
 cmd guilt pop
 fixup_time_info prepend
-cmd guilt push
+cmd guilt push 2>&1 | grep -v "fatal: unrecognized input"
 
 cmd list_files
 
Index: guilt-0.35/regression/t-028.sh
===================================================================
--- guilt-0.35.orig/regression/t-028.sh
+++ guilt-0.35/regression/t-028.sh
@@ -19,7 +19,7 @@ cmd guilt push -a
 cmd guilt new -s -m "blah blah blah" patch-with-some-desc
 cmd guilt pop
 fixup_time_info patch-with-some-desc
-cmd guilt push
+cmd guilt push 2>&1 | grep -v "fatal: unrecognized input"
 
 cmd list_files
 
Index: guilt-0.35/regression/t-051.sh
===================================================================
--- guilt-0.35.orig/regression/t-051.sh
+++ guilt-0.35/regression/t-051.sh
@@ -25,13 +25,13 @@ function std_tests
 # create a couple of dummy (empty) patches strategically placed thoughout
 # the series
 cmd guilt new first
-cmd guilt push modify
+cmd guilt push modify 2>&1 | grep -v "fatal: unrecognized input"
 cmd guilt new second
-cmd guilt push add
+cmd guilt push add 2>&1 | grep -v "fatal: unrecognized input"
 cmd guilt new third
-cmd guilt push remove
+cmd guilt push remove 2>&1 | grep -v "fatal: unrecognized input"
 cmd guilt new fourth
-cmd guilt push mode
+cmd guilt push mode 2>&1 | grep -v "fatal: unrecognized input"
 cmd guilt new fifth
 cmd guilt pop -a
 fixup_time_info first
@@ -48,7 +48,7 @@ p=first
 
 # no guarded patches; no guards selected
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -56,7 +56,7 @@ cmd list_files
 # no guarded patches; one guard selected
 cmd guilt select foo
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -65,7 +65,7 @@ cmd list_files
 cmd guilt select -n
 cmd guilt guard $p +foo
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -73,7 +73,7 @@ cmd list_files
 # one positive guarded patch; that guard selected
 cmd guilt select foo
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -82,7 +82,7 @@ cmd list_files
 cmd guilt select -n
 cmd guilt select bar
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -92,7 +92,7 @@ cmd guilt select -n
 cmd guilt guard -l
 cmd guilt guard $p -foo
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -100,7 +100,7 @@ cmd list_files
 # one negative guarded patch; that guard selected
 cmd guilt select foo
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files
@@ -109,7 +109,7 @@ cmd list_files
 cmd guilt select -n
 cmd guilt select bar
 cmd guilt guard -l
-cmd guilt push -a
+cmd guilt push -a 2>&1 | grep -v "fatal: unrecognized input"
 std_tests
 cmd guilt pop -a
 cmd list_files 
